### ISSUE TITLE
Project Creation wizard does not support "Enter path to installed PhoneGap" PhoneGap 1.4.0

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#66]
```
Workaround:

Rename lib/android -> lib/Android
Rename lib/android/example -> lib/android/Sample
Point to {installdir}/lib




```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 31 Jan 2012 at 7:10

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 2 Feb 2012 at 12:02
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
1.2.7 has been updated to handle the new directory structure
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
- Changed state: **Fixed**
